packages/tests/test/reya_common/trade/PassivePool.fork.c.sol

```diff
@@ -255,4 +266,288 @@ contract PassivePoolForkCheck is BaseReyaForkTest {
         // withdraw 100 sUSDe from account
         executePeripheryWithdrawMA(user, userPk, 1, accountId, sec.susde, 100e18, sec.mainChainId);
     }
+
+    function check_PassivePoolWithDeusd() public {
+        mockFreshPrices();
+
+        (CollateralConfig memory collateralConfig, ParentCollateralConfig memory parentCollateralConfig,) =
+            ICoreProxy(sec.core).getCollateralConfig(1, sec.deusd);
+
+        vm.prank(sec.multisig);
+        collateralConfig.cap = type(uint256).max;
+        ICoreProxy(sec.core).setCollateralConfig(1, sec.deusd, collateralConfig, parentCollateralConfig);
+
+        (address user, uint256 userPk) = makeAddrAndKey("user");
+
+        uint256 sharePrice0 = IPassivePoolProxy(sec.pool).getSharePrice(sec.passivePoolId);
+
+        // add 3000 deUSD to the passive pool directly
+        deal(sec.deusd, address(sec.periphery), 3000e18);
+        mockBridgedAmount(dec.socketExecutionHelper[sec.deusd], 3000e18);
+        vm.prank(dec.socketExecutionHelper[sec.deusd]);
+        IPeripheryProxy(sec.periphery).depositExistingMA(
+            DepositExistingMAInputs({ accountId: sec.passivePoolAccountId, token: address(sec.deusd) })
+        );
+
+        // check that the new 3000 deUSD does not influence the share price
+        uint256 sharePrice1 = IPassivePoolProxy(sec.pool).getSharePrice(sec.passivePoolId);
+        assertApproxEqRelDecimal(sharePrice1, sharePrice0, 0.005e18, 18);
+
+        // make sure that the passive pool deposit works
+        deal(sec.usdc, sec.periphery, 10e6);
+        vm.prank(dec.socketExecutionHelper[sec.usdc]);
+        vm.mockCall(
+            dec.socketExecutionHelper[sec.usdc],
+            abi.encodeCall(ISocketExecutionHelper.bridgeAmount, ()),
+            abi.encode(10e6)
+        );
+
+        IPeripheryProxy(sec.periphery).depositPassivePool(
```

Comment:
will be good to also add new periphery functions depositPassivePoolV2 and withdrawPassivePoolV2